fe/src/components/Sidebar/SidebarItemLink.tsx

```diff
@@ -1,32 +1,29 @@
-import { Button } from '@chakra-ui/react';
-import React, { ReactElement } from 'react';
-import {
-  Link as RouteLink,
-} from 'react-router-dom';
+import { Box, Button, Text } from '@chakra-ui/react';
+import { IconName } from '@fortawesome/fontawesome-common-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { Link as RouteLink } from 'react-router-dom';
 
 interface SidebarItemLinkProps {
-  icon: ReactElement<string | undefined>;
+  icon: string;
```

Comment:
I guess this will work, but some typing for the icons would be nice. I however have no idea how that could be established 🤔

Reply:
Yeah. Will look into it if its possible to do

Reply:
if you mean this. It is automatically available: 
![image](https://user-images.githubusercontent.com/48998818/118837965-f39ea900-b8cd-11eb-9fae-bac9619d8a24.png)

but now you can access icons in here too:
![image](https://user-images.githubusercontent.com/48998818/118838063-0add9680-b8ce-11eb-890b-67431d3a50df.png)

Reply:
Wait, how can it have autocomplete if it's of type `string`?